### PR TITLE
Doc Updates and Edits

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 1]
```diff
@@ -1,112 +1,161 @@
...
+
+. In the *Secret Groups* list view, select the secret group to which to add a keystore. 
+. Select *Keystore* in the menu on the left, and click *Add Keystore*. 
+. In the *Name* field, enter a name for your keystore.
+. In the *Type* field, select the keystore type from the drop-down menu.
 +
-Supported types are:
+Supported types include the following:
++
+* Privacy Enhanced Mail (PEM) - Base64 encoded ASCII file with a cer, crt, or pem extension.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: `.cer`, `.crt`, or `.pem` extension (for translation makes it easy to know these aren't words they don't recognize).

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 2]
```diff
@@ -1,112 +1,161 @@
...
+* Java Keystore (JKS) - Repository for authorization or public key certificates. The JKS keystore type does not store secret keys.
+* PKCS12 - Stores server and intermediate certificates in an archive file format. The PKCS12 keystore type does not store secret keys.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:39]: Stores a server (any reason to leave out the article?)

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 3]
```diff
@@ -1,112 +1,161 @@
...
+* Java Cryptography Extension KeyStore (JCEKS) - Stores server and intermediate certificates as well as secret keys.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:40]: Stores a server (if same as line 39)

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 4]
```diff
@@ -1,112 +1,161 @@
...
-. In the Truststore File field, click Choose File, and select your truststore file to upload.
+** *Certificate File* - Click *Choose File* to locate and upload the PEM certificate file.  
+** *Key File* - Click *Choose File* to locate and upload the PEM formatted file that contains the private key for the certificate. 
+** *Key Passphrase* - Enter the word or phrase that protects the private key. 
+** *CA Path Certificate File* (Optional) - Click *Choose File* to locate and upload the certificate signed by a certification authority   
+** *Override Expiration Date* (Optional) - Select the date to override the default expiration date of the certificate.
+.. To add a *JKS*, *PKCS12*, or *JCEKS* type keystore, you must provide the following: 
++
+** *Keystore File* - Click *Choose File* to locate and upload the keystore file to use.
+** *Keystore Passphrase* - Enter the word or phrase that protects the Keystore.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: lowercase for keystore at the end of the sentence?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 5]
```diff
@@ -1,112 +1,161 @@
...
+** *Alias* - The alias referenced by the Java program.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: What Java program?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 6]
```diff
@@ -1,112 +1,161 @@
...
-If you are uploading a JKS, PKCS12, or JCEKS truststore file, you also need to provide the passphrase for this truststore.
+== Add a Truststore
 
-== Adding a Certificate
+A truststore includes only public certificates of trusted servers. 
 
-. In the secret group to which you want to add this secret, select Certificate, and click the Add Certificate button.
-. In the Name field, write a name for your certificate.
-. In the Type field, select the certificate type from the drop-down menu.
+. In the *Secret Groups* list view, select the secret group to which to add a truststore.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: "to which to add" is a little hard to read. Is there a simpler way to describe? Maybe "select the secret group to receive a new truststore." ?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 7]
```diff
@@ -1,112 +1,161 @@
...
+* *Name* - Enter a name for your shared secret.
+* *Type* - Select the shared secret type from the drop-down menu.
+ ** *Username Password* - Provide a username and password.
+ ** *Symmetric Key* - Provide a Base64 string containing symmetric key.
+ ** *S3 Credential* - Provide the access key ID and the secret access key to an S3 bucket.
+ ** *Blob* - Provide a base64-encoded value.
+. Click *Save*. 
 
-== Adding a Certificate Revocation List Distribution Configuration
+== Add a Certificate Revocation List Distributor Configuration
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: I try to break up noun phrases when they get too long. Add a List of Certificate Revocations for CRL Distribution Points (?).

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 8]
```diff
@@ -1,112 +1,161 @@
...
-. In the secret group to which you want to add this secret, select CRL Distribution Point, and click the Add CRL button.
-. In the name field, write a name for your CRL distribution point.
-. In the Distributor Certificate field, select your CRL distributor from the drop-down menu. +
-//_COMBAK: How to populate this drop-down.
-. In the CA Certificate field, select the CA certificate to query against the CRL distributor from the drop-down menu. +
-//_COMBAK: This can be a URL, FTP, and LDAP?
-. In the Frequency field, determine the interval in minutes to query the CRL distributor.
-. In the Complete CRL Issuer URL, add the CRL issuer URL. +
-Additionally, you can add a Delta CRL in the Delta CRL Issuer URL field.
+Add a list of certificates that were revoked by certificate authorities before their expiration dates.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Where do they obtain this list?

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 9]
```diff
@@ -1,112 +1,161 @@
...
 
-== Adding a TLS Context
+. In the *Secret Groups* list view, select the secret group to which to add a Certificate List Distributor (CRL) configuration.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:126]: Do we explain anywhere what "Distributor configuration" means?

**Review by Mulesoft-Mysti — APPROVED** [review]
Assume you tested the changed links. Also, at some point it would be great to explain why to do or not do each thing (unless users would already know). Understood if not in scope for this edit.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 9 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc:126]: It's Step 3

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 8 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Good question.

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc:39]: I don't think it needs the "a." 

Stores the server [certificates] and intermediate certificates...

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc (hunk 5 above)
Comment (valkyrie69) [reply modules/ROOT/pages/asm-secret-group-creation-task.adoc]: Changed to: 
The alias used to access the keystore entries (key and trusted certificate entries).

**Review by Mulesoft-Mysti — APPROVED** [review]

**Pull request merged**